Makefile

```diff
@@ -44,11 +44,11 @@ vet:
 	go vet ./...
```

Comment:
Could you also remove `tools/make-hsm-ee/make-hsm-ee` from .gitignore?

Reply:
yes!